### PR TITLE
[Misc] Rename number_of_edges and number_of_nodes to num_edges and num_nodes in examples.

### PR DESCRIPTION
**Pull request by frozenbugs**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: examples/mxnet/gat/train.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def main(args):
     test_mask = mx.nd.array(np.nonzero(test_mask.asnumpy())[0], ctx=ctx)
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/gat/train.py:73]: I doubt if this even works as `data` no longer has the `.graph` API.

File: examples/mxnet/gcn/gcn_concat.py [hunk 2]
```diff
@@ -95,7 +95,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/gcn/gcn_concat.py:98]: ditto

File: examples/mxnet/gcn/train.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/gcn/train.py:50]: ditto

File: examples/mxnet/graphsage/main.py [hunk 4]
```diff
@@ -112,7 +112,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/graphsage/main.py:115]: ditto

File: examples/mxnet/monet/citation.py [hunk 5]
```diff
@@ -94,7 +94,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/monet/citation.py:97]: ditto

File: examples/mxnet/sgc/sgc.py [hunk 6]
```diff
@@ -57,7 +57,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/sgc/sgc.py:60]: ditto

File: examples/mxnet/tagcn/train.py [hunk 7]
```diff
@@ -49,7 +49,7 @@ def main(args):
     test_mask = g.ndata["test_mask"]
     in_feats = features.shape[1]
     n_classes = data.num_labels
-    n_edges = data.graph.number_of_edges()
+    n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/mxnet/tagcn/train.py:52]: ditto

File: examples/tensorflow/gcn/gcn_builtin.py [hunk 8]
```diff
@@ -115,7 +115,7 @@ def main(args):
         test_mask = g.ndata["test_mask"]
         in_feats = features.shape[1]
         n_classes = data.num_labels
-        n_edges = data.graph.number_of_edges()
+        n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/tensorflow/gcn/gcn_builtin.py:118]: ditto

File: examples/tensorflow/gcn/gcn_mp.py [hunk 9]
```diff
@@ -122,7 +122,7 @@ def main(args):
         test_mask = g.ndata["test_mask"]
         in_feats = features.shape[1]
         n_classes = data.num_labels
-        n_edges = data.graph.number_of_edges()
+        n_edges = data.graph.num_edges()
```
Comment (mufeili) [inline examples/tensorflow/gcn/gcn_mp.py:125]: ditto

**Review by mufeili — COMMENTED** [review]
done a pass

**Comment by frozenbugs** [comment]
Closed this PR since they are all outdated MXNet and TF examples.

**Pull request closed without merging**